css/themes/default/jquery.mobile.theme.css

```diff
@@ -611,3 +613,22 @@ button[disabled],
 	outline-width: 1px;
 	outline-style: inherit;
 }
+
+.ui-accordion-header {
+	background: #e6e6e6;
+	border: 1px solid #d3d3d3;
+	color: #555555;
+}
+
+.ui-accordion-content {
+	border: 1px solid #aaaaaa;
+	background: #ffffff;
+	color: #222222;
+}
+
+.ui-accordion-header-active,
+.ui-accordion-header.ui-state-active  {
+	border: 1px solid #aaaaaa;
+	background: #ffffff;
+	color: #212121;
+}
```

Comment:
This should use the existing active colors https://github.com/jquery/jquery-mobile/blob/1.5-dev/css/themes/default/jquery.mobile.theme.css#L508 and https://github.com/jquery/jquery-mobile/blob/1.5-dev/css/themes/default/jquery.mobile.theme.css#L341